k4MarlinWrapper/src/components/Lcio2EDM4hep.cpp

```diff
@@ -62,9 +61,13 @@ StatusCode Lcio2EDM4hepTool::finalize() { return AlgTool::finalize(); }
 // Check if a collection was already registered to skip it
 // **********************************
 bool Lcio2EDM4hepTool::collectionExist(const std::string& collection_name) {
-  auto collections = m_podioDataSvc->getEventFrame().getAvailableCollections();
+  std::vector<std::string> collections;
+  //TODO:
```

Comment:
Is this still a real TODO or is it a left over?

Reply:
This is now done after adding the new library and the function to get collections from the store when using IOSvc.